<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-G790OGBU" name="com.menumitratCommonAPITestScript.sectionListViewTestScript" tests="3" failures="0" timestamp="6 May 2025 05:35:42 GMT" time="2.661" errors="1">
  <testcase name="sectionListViewNegativeTest" time="0.651" classname="com.menumitratCommonAPITestScript.sectionListViewTestScript"/>
  <testcase name="verifySectionListViewUsingValidInputData" time="1.993" classname="com.menumitratCommonAPITestScript.sectionListViewTestScript"/>
  <testcase name="sectionListViewNegativeTest" time="0.017" classname="com.menumitratCommonAPITestScript.sectionListViewTestScript">
    <error type="com.menumitra.utilityclass.customException" message="Error in section list view negative test: JSONObject[&quot;outlet_id&quot;] is not a int (class java.lang.String : @).">
      <![CDATA[com.menumitra.utilityclass.customException: Error in section list view negative test: JSONObject["outlet_id"] is not a int (class java.lang.String : @).
	at com.menumitratCommonAPITestScript.sectionListViewTestScript.sectionListViewNegativeTest(sectionListViewTestScript.java:452)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- sectionListViewNegativeTest -->
</testsuite> <!-- com.menumitratCommonAPITestScript.sectionListViewTestScript -->
